src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,316 @@
+        i18n: {
+            defaultLocale: "en",
+            locales: ["en", "es", "pt_BR"]
+        }
+    }
+})
+```
+
+Organize your content folders by locale, including your `defaultLocale` and `src/pages/index.astro` will now automatically default to the `index.astro` file of your default language.
+
+```
+├── src
+│   ├── pages
+│   │   ├── en
+│   │   │   ├── about.astro
+│   │   │   ├── index.astro
+│   │   ├── es
+│   │   │   ├── about.astro
+│   │   │   ├── index.astro
+│   │   ├── pt_BR
+│   │   │   ├── about.astro
+│   │   │   ├── index.astro
+│   ├── index.astro
+```
+
+Compute relative URLs for your links with `getLocaleRelativeURL` from the new `astro:i18n` module:
+
+```astro
+---
+import {getLocaleRelativeUrl} from "astro:i18n";
+const aboutUrl = getLocaleRelativeUrl("pt_Br", "about");
+---
+<p>Learn more <a href={aboutURL}>About</a> this site!</p>
+```
+
+Enabling i18n routing also provides 
+
+
+## Glossary
+
+The current page will use some specific terminology related to internationalization:
+- Locale: a code that represents a language.
+- Localized folder: a folder/directory that is named after a locale.
+
+*/}
+
+
+<Since v="3.5.0" />
+
+Astro's experimental i18n routing allows you to add your multilingual content with support for configuring a default language and URL path convention, computing relative page URLs, and accepting preferred languages provided by your visitor's browser.
+
+This routing API helps you generate, use, and verify the URLs that your multi-language site produces. Check back and update regularly for the latest changes as this API continues to develop!
+
+
+### Configure i18n routing
+
+1. Enable the experimental routing option by adding an `i18n` object to your Astro configuration with a [default location (`defaultLocale`) and a list of all languages to support (`locales`)](#defaultlocale-and-locales):
+
+   ```js title="astro.config.mjs"
+   import { defineConfig } from "astro/config"
+   export default defineConfig({
+     experimental: {
+       i18n: {
+         defaultLocale: "en",
+         locales: ["en", "es", "pt_BR"]
+       }
+     }
+   })
+   ```
+  
+    You can also specify a [`routingStrategy`](#routing-strategy) to customize the URLs for your default language.
+
+
+2. Organize your content folders with localized content by language, including your `defaultLocale`. Your folder names must match your lthe items in `locales` exactly. `src/pages/index.astro` will now automatically redirect to the `index.astro` file of your default language.
+    <FileTree>
+    - src
+      - pages
+        - en
+          - about.astro
+          - index.astro
+        - es
+          - about.astro
+          - index.astro
+        - pt_BR
+          - about.astro
+          - index.astro
+        - index.astro  (redirects to the route created for `/en/index.astro`)
+    </FileTree>
+
+    :::note
+    The localised folders do not need to be at the root of the `/pages/` folder.
+    
```

Comment:
Typo: `localised` -> `localized`
```suggestion
    The localized folders do not need to be at the root of the `/pages/` folder.
    
```

Reply:
This is actually valid, are the Docs specifically US English or UK English? I guess other places here use "localize", but my Mac set to UK English autocorrects with an `s`.

Reply:
they are specifically US English!